front/app/containers/Admin/projects/all/index.tsx

```diff
@@ -91,10 +91,22 @@ const AdminProjectsList = memo(({ className }: Props) => {
 
   const userIsAdmin = isAdmin(authUser);
 
+  // Fetch the admin publications to show in the 'Your projects' tab,
+  // including the (unexpanded) folders the user is a moderator of.
   const { data: moderatedAdminPublications } = useAdminPublications({
     publicationStatusFilter: ['published', 'draft', 'archived'],
-    onlyProjects: true,
     filter_is_moderator_of: true,
+    exclude_projects_in_included_folders: true,
```

Comment:
How is this different from using the `depth: 0` query param (called `rootLevelOnly` in the FE/if you want to use it with `useAdminPublications`)?

Reply:
I seem to get the same results.

Reply:
Are you seeing only folders you moderate and not folders you do not moderate?

If you are project moderator of a project in a folder you do not moderate, do you see the project?

Reply:
I can see a difference now! With `rootLevelOnly`/`depth: 0` I wouldn't see a project I'm managing inside a folder (if I'm also not a manager of its folder). With the new param I do.

Reply:
I've improved the comments in this file, to help explain.